analyzer_test.go

```diff
@@ -13,6 +13,37 @@ import (
 	"golang.org/x/tools/go/analysis/passes/inspect"
 )
 
+func makeAnalyzer() *analysis.Analyzer {
+	cnf := Config{
+		RequiredAlias: make(map[string]string),
+	}
+	return &analysis.Analyzer{
+		Flags: flags(&cnf),
+		Run: func(pass *analysis.Pass) (interface{}, error) {
+			return runWithConfig(&cnf, pass)
+		},
+		Requires: []*analysis.Analyzer{inspect.Analyzer},
+	}
+}
+
+func TestIncorrectFlags(t *testing.T) {
+	t.Run("wrong usage", func(t *testing.T) {
```

Comment:
don't need the t.Run since there's only one test here :)

Reply:
your'e right